internal/net/http/middleware/timeout_test.go

```diff
@@ -112,7 +112,7 @@ func TestWrap(t *testing.T) {
 			}
 		}(),
 		func() test {
-			wantErr := errors.Errorf("faild")
+			wantErr := errors.New("faild")
```

Comment:
can you explain why `New()` works but `Errorf` doesn't work?

Reply:
if argument of `Errorf` is nil, `Errorf` function returns nil.
The following is details.
https://github.com/vdaas/vald/blob/master/internal/errors/errors.go#L98

Reply:
ohhhh....thank you for your explanation. I missed the implementation detail.